modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -1,503 +1,302 @@
-:description: Information about how to use vector indexes to perform approximate nearest neighbor search in Neo4j.
+:description: Information about creating, querying, and deleting vector indexes with Cypher.
 :page-role: new-5.13
-:fn-hnsw: footnote:hnsw[http://dx.doi.org/10.1109/TPAMI.2018.2889473[Efficient and Robust Approximate Nearest Neighbor Search Using Hierarchical Navigable Small World Graphs] -- Yury A. Malkov and Dmitry A. Yashunin (preprint: https://arxiv.org/abs/1603.09320:[arXiv:1603.09320])]
-:fn-ieee-754: footnote:ieee-754[https://doi.org/10.1109/IEEESTD.2019.8766229[IEEE Standard for Floating-Point Arithmetic]]
-:fn-provider-differences: footnote:provider-differences[xref:indexes-vector-provider-differences[]]
-:page-role: new-5.13
-:link-operations-manual: link:{neo4j-docs-base-uri}/operations-manual/{page-version}
-:link-procedures-reference: {link-operations-manual}/reference/procedures/
-:ieee-754: IEEE 754{fn-ieee-754}
```

Comment:
I see you've removed the linking to IEEE 754, I think it would be useful to link it once at least somewhere, even if it is to the Wikipedia article rather than the standard itself. Perhaps in the similarity section.

Reply:
I also see you've rendered it as `IEEE-754` rather than `IEEE 754` throughout the document. It should be rendered with a space not a dash as per the IEEE convention, the dash should be used only when specifying a publishing year (if you need to be specific about a revision)
IEEE 754
IEEE 754-2019 (current version)